tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/DefaultTaggedMetricRegistry.java

```diff
@@ -36,6 +36,7 @@
     private static final TaggedMetricRegistry DEFAULT = new DefaultTaggedMetricRegistry();
 
     private final Map<MetricName, Metric> registry = new ConcurrentHashMap<>();
+    private final Map<Map<String, String>, TaggedMetricSet> taggedRegistries = new ConcurrentHashMap<>();
```

Comment:
probably not _quite right_, since will want to shove probably two MetricSets in for each AtlasDB, also probably shouldn't be passing in an empty map here. Maybe just a Set of Pair type structure is the right thing here.